libs/fetcher.js

```diff
@@ -303,7 +318,10 @@ Fetcher.middleware = function () {
                         res.status(errResponse.statusCode).json(errResponse.output);
                         return;
                     }
-                    res.status(meta.statusCode || 200).json(data);
+                    res.status(meta.statusCode || 200).json({
```

Comment:
Not really.

This PR also refactors the client fetchr so it parses out the `data` and `meta` separately. So as long as the middleware is only accessed by fetchr client calls and not directly via urls( which we told people to stop doing in the [last minor](https://github.com/yahoo/fetchr/releases/tag/v0.5.0)) this isn't a breaking change.